src/app/components/login/login.rs

```diff
@@ -9,12 +11,37 @@ use crate::app::AppEvent;
 
 use super::LoginModel;
 
+const KEY_RETURN: u32 = 65293;
```

Comment:
Have you tried using [keyval_from_name](https://gtk-rs.org/docs/gdk/fn.keyval_from_name.html)? Might handle the keycode conversion subtleties for us :)

Reply:
I'll look into it